config.py

```diff
@@ -41,20 +48,12 @@ class Config:
     LANGUAGES = ['en-GB', 'fr']
 
     UPLOADED_PATH = os.path.join(basedir, 'uploads')
+    ALLOWED_EXTENSIONS = {'txt', 'pdf', 'png', 'jpg', 'jpeg', 'gif', 'ima', 'dcm'}
+
     DROPZONE_MAX_FILE_SIZE = 3
-    DROPZONE_MAX_FILES = 20
+    DROPZONE_PARALLEL_UPLOADS = 2
+    DROPZONE_MAX_FILES = 100
```

Comment:
adjusted DropZone config to allow for more robust file upload that suits the needs of our application